lib/Runtime/Debug/TTSerialize.cpp

```diff
@@ -894,7 +894,7 @@ namespace TTD
         // Null-terminate the list before we try to use the buffer as a string.
         charList.Add(_u('\0'));
 
-        bool likelyint; //we don't care about this just want to know that it is convertable to a number
+        LikelyNumberType likelyint; //we don't care about this just want to know that it is convertable to a number
```

Comment:
Can you confirm that TTD tests work with bigint and that you dont need to do any special handling? (It's ok if you defer any extra work on this front to a future PR but just making sure you validated that scenario)

Reply:
Time travel debugging.

Reply:
Yup 😄 @duongnhn can you sync up with @kfarnung tomorrow, he can show you how to run those tests.